Makefile

```diff
@@ -1,4 +1,24 @@
+
+IMAGE=segment/mongodb-source
+VERSION=v0.1.3-beta
+
+save-deps:
+	godep save
+
+run:
+	go run main.go mongo.go collection.go config.go description.go $(ARGS)
+
 build:
```

Comment:
we should use either
- godep build; or
- govendor and document go1.5+ (w/ flag enabled; by default on 1.6)